src/parameterized_tendencies/radiation/RRTMGPInterface.jl

```diff
@@ -1118,6 +1119,27 @@ function update_boundary_layer_cloud!(cloud_state)
     @views cloud_state.cld_frac[end, :] .= cloud_state.cld_frac[end - 1, :]
 end
 
+update_boundary_layer_aerosol!(::GrayRadiation, _) = nothing
+update_boundary_layer_aerosol!(
+    ::Union{
+        ClearSkyRadiation,
+        AllSkyRadiation,
+        AllSkyRadiationWithClearSkyDiagnostics,
+    },
```

Comment:
Since this is everything that's not `GrayRadiation`, you don't need to specify this `Union` type here. The method above will catch `GrayRadiation`, and this can just catch everything else.